src/applications/personalization/dashboard-2/utils/prescriptions.js

```diff
@@ -0,0 +1,355 @@
+export const prescriptions = {
```

Comment:
👍 just curious where you found these :)

Reply:
I copied the prescription's response from staging user +377